MCA/api/routes/staff/index.ts

```diff
@@ -30,4 +34,51 @@ staffRouter.get("/categories/:year", validatePhaseYear, async (ctx) => {
     ctx.body = categories.map(x => x.getInfo());
 });
 
+// Endpoint for granting direct MCA nom/vote access to users
+staffRouter.post("/grant/:year", validatePhaseYear, async (ctx) => {
```

Comment:
need logging or further restriction

Reply:
resolved in https://github.com/Corsace/Corsace/pull/158/commits/f152582a794f90550e915e7a4b2858de15cad27c